### ISSUE TITLE
[TODO] investigate import errors in various locations

### ISSUE THREAD

**Issue by ITProKyle** [onicagroup/runway#432]
#429 moved the stacker shim to resolve some import errors and as a result causing/uncovering some other import errors.

**These errors are not present during normal execution (CLI invocation). They arose in our test suite where `conftest.py` imports portions of runway outside of normal execution.**

This error was resolved by changing this import:
```python
from runway.config import Config
from runway.core.components import DeployEnvironment
 ```

to this and consuming the components inline (e.g. `runway.config.Config()`):
```python
import runway
```

Looking at the error, it appears to be a cyclic import issue involving the Runway config, CFNgin lookups, hooks, variables, and exceptions (#430). To resolve this, we will need to shuffle around where things are defined which will cause things to break. notably any hook or custom module that handles exceptions or still implements in-hook variable resolution.

Since this does not impact executing Runway via the CLI (the primary intent of this project) I don't perceive this as a critical issue and can be fixed in the next scheduled major release without expediting timeline.

**Comment by ITProKyle** [comment]
This appears to have been resolved by #439 and #440 but will need to test further once #429 has been merged and incorperated into the v2 branch.

These fixes can't be applied to v1.

**Comment by ITProKyle** [comment]
Updated the v2 branch after #429 was merged into master. Removed the relocated stacker shim and reverted the changes to conftest (#452). The issue did not reoccur. Closing as solved by the linked PRs.
